### ISSUE TITLE
2020-11

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#20]
# Oct - 2021

## 11.22 - 11.26
| Title |
| ----- |
An Optimization Framework for Federated Edge Learning
Client Selection in Federated Learning based on Gradients Importance
Decentralized Deep Learning for Multi-Access Edge Computing: A Survey on Communication Efficiency and Trustworthiness
Efficient Secure Aggregation Based on SHPRG For Federated Learning
Fed2: Feature-Aligned Federated Learning
FedCV: A Federated Learning Framework for Diverse Computer Vision Tasks
FedDropoutAvg: Generalizable federated learning for histopathology image classification
Fed-EINI: An Efficient and Interpretable Inference Framework for Decision Tree Ensembles in Federated Learning
Federated Dynamic Neural Network for Deep MIMO Detection
Federated Gaussian Process: Convergence, Automatic Personalization and Multi-fidelity Modeling
FLIX: A Simple and Communication-Efficient Alternative to Local Methods in Federated Learning
FL-Market: Trading Private Models in Federated Learning
Forget-SVGD: Particle-Based Bayesian Federated Unlearning
Hierarchical Federated Learning based Anomaly Detection using Digital Twins for Smart Healthcare
Incentive Mechanisms for Federated Learning: From Economic and Game Theoretic Perspective
Intracranial Hemorrhage Detection Using Neural Network Based Methods With Federated Learning
Jointly Learning from Decentralized (Federated) and Centralized Data to Mitigate Distribution Shift
Local Learning Matters: Rethinking Data Heterogeneity in Federated Learning
Non-IID data and Continual Learning processes in Federated Learning: A long road ahead
On-Board Federated Learning for Dense LEO Constellations
ProxyFL: Decentralized Federated Learning through Proxy Model Sharing
Resource-Aware Asynchronous Online Federated Learning for Nonlinear Regression
Robust and Differentially Private Mean Estimation
Semantic-Aware Collaborative Deep Reinforcement Learning Over Wireless Cellular Networks

## 11.15 - 11.19
| Title |
| ----- |
A Novel Optimized Asynchronous Federated Learning Framework	
Advancing COVID-19 Diagnosis with Privacy-Preserving Collaboration in Artificial Intelligence	
An Energy Consumption Model for Electrical Vehicle Networks via Extended Federated-learning	
An Expectation-Maximization Perspective on Federated Learning	
An Expectation-Maximization Perspective on Federated Learning	
Broadband Digital Over-the-Air Computation for Asynchronous Federated Edge Learning	
Client Selection in Federated Learning based on Gradients Importance	
Data privacy protection in microscopic image analysis for material data mining	
Differentially Private Federated Learning on Heterogeneous Data	
Distributed Unsupervised Visual Representation Learning with Fused Features	
DNN gradient lossless compression: Can GenNorm be the answer?	
EdgeML: Towards Network-Accelerated Federated Learning over Wireless Edge	
FedCG: Leverage Conditional GAN for Protecting Privacy and Maintaining Competitive Performance in Federated Learning	
FedCostWAvg: A new averaging for better Federated Learning	
Federated Learning for Malware Detection in IoT Devices	
Federated Learning for Smart Healthcare: A Survey	
Federated Learning with Domain Generalization	
Federated Noisy Client Learning	
Federated Social Recommendation with Graph Neural Network	
FLSys: Toward an Open Ecosystem for Federated Learning Mobile Apps	
FLSys: Toward an Open Ecosystem for FederatedLearning Mobile Apps	
Genetic CFL: Optimization of Hyper-Parameters in Clustered Federated Learning	
HADFL: Heterogeneity-aware Decentralized Federated Learning Framework	
Incentive Mechanisms for Federated Learning: From Economic and Game Theoretic Perspective	
Inference-Time Personalized Federated Learning	
Local Model Poisoning Attacks to Byzantine-Robust Federated Learning	
On the Tradeoff between Energy, Precision, and Accuracy in Federated Quantized Neural Networks	
Over-the-Air Federated Learning with Retransmissions (Extended Version)	
Over-the-Air Federated Learning with Retransmissions (Extended Version)	
Personalized Federated Learning through Local Memorization	
Privacy-preserving Federated Adversarial Domain Adaption over Feature Groups for Interpretability	
Satellite Based Computing Networks with Federated Learning	
Secure Federated Learning for Residential Short Term Load Forecasting	
Secure Linear Aggregation Using Decentralized Threshold Additive Homomorphic Encryption For Federated Learning	
Understanding Training-Data Leakage from Gradients in Neural Networks for Image Classification	
Understanding Training-Data Leakage from Gradients in Neural Networks for Image Classification	
Vehicular Cooperative Perception Through Action Branching and Federated Reinforcement Learning	
Wyner-Ziv Gradient Compression for Federated Learning	

## 11.08 - 11.12
| Title |
| ----- |
Device-Independent-Quantum-Randomness-Enhanced Zero-Knowledge Proof
DP-REC: Private & Communication-Efficient Federated Learning
Edge-Native Intelligence for 6G Communications Driven by Federated Learning: A Survey of Trends and Challenges
Eluding Secure Aggregation in Federated Learning via Model Inconsistency
Fairness, Integrity, and Privacy in a Scalable Blockchain-based Federated Learning System
Feature Concepts for Data Federative Innovations
Federated Learning Based on Dynamic Regularization
Federated Learning for Internet of Things: Applications, Challenges, and Opportunities
Federated Learning with Hyperparameter-based Clustering for Electrical Load Forecasting
FedGreen: Federated Learning with Fine-Grained Gradient Compression for Green Mobile Edge Computing
Flatee: Federated Learning Across Trusted Execution Environments
Hierarchical Bayesian Bandits
Linear Speedup in Personalized Collaborative Learning
Papaya: Practical, Private, and Scalable Federated Learning
Power Allocation for Wireless Federated Learning using Graph Neural Networks
STFL: A Temporal-Spatial Federated Learning Framework for Graph Neural Networks
The Internet of Federated Things (IoFT): A Vision for the Future and In-depth Survey of Data-driven Approaches for Federated Learning
Towards Privacy-Preserving Affect Recognition: A Two-Level Deep Learning Architecture
Transmission Power Control for Over-the-Air Federated Averaging at Network Edge
Unified Group Fairness on Federated Learning

## 11.01 - 11.05
| Title |
| ----- |
A Cyber Threat Intelligence Sharing Scheme based on Federated Learning for Network Intrusion Detection	
A Federated Learning Framework for Smart Grids: Securing Power Traces in Collaborative Learning	
A Personalized Federated Learning Algorithm: an Application in Anomaly Detection	
A Survey of Fairness-Aware Federated Learning	
Basis Matters: Better Communication-Efficient Second Order Methods for Federated Learning	
CAFE: Catastrophic Data Leakage in Vertical Federated Learning	
Data Selection for Efficient Model Update in Federated Learning	
Decentralized Personalized Federated Min-Max Problems	
Distributed Learning over a Wireless Network with FSK-Based Majority Vote	
Distributed Reinforcement Learning for Privacy-Preserving Dynamic Edge Caching	
DQRE-SCnet: A novel hybrid approach for selecting users in Federated Learning with Deep-Q-Reinforcement Learning based on Spectral Clustering	
DVFL: A Vertical Federated Learning Method for Dynamic Data	
Federated Expectation Maximization with heterogeneity mitigation and variance reduction	
Federated Learning Attacks Revisited: A Critical Discussion of Gaps, Assumptions, and Evaluation Setups	
Federated Learning from Small Datasets	
Federated Split Vision Transformer for COVID-19CXR Diagnosis using Task-Agnostic Training	
FedFly: Towards Migration in Edge-based Distributed Federated Learning	
FedFm: Towards a Robust Federated Learning Approach For Fault Mitigation at the Edge Nodes	
FedGraph: Federated Graph Learning with Intelligent Sampling	
FedLess: Secure and Scalable Federated Learning Using Serverless Computing	
FedScale: Benchmarking Model and System Performance of Federated Learning at Scale	
Implicit Model Specialization through DAG-based Decentralized Federated Learning	
Learning Federated Representations and Recommendations with Limited Negatives	
Memory-based Optimization Methods for Model-Agnostic Meta-Learning	
Optimal Accounting of Differential Privacy via Characteristic Function	
Parameterized Knowledge Transfer for Personalized Federated Learning	
Practical and Light-weight Secure Aggregation for Federated Submodel Learning	
Privacy attacks for automatic speech recognition acoustic models in a federated learning framework	
Privacy-Preserving Communication-Efficient Federated Multi-Armed Bandits	
Private Federated Learning Without a Trusted Server: Optimal Algorithms for Convex Losses	
Resource-Efficient Federated Learning	
Robust Federated Learning via Over-The-Air Computation	
Sharp Bounds for Federated Averaging (Local SGD) and Continuous Perspective	
TEE-based Selective Testing of Local Workers in Federated Learning Systems
